### PR TITLE
update deps and fix typo in README

### PR DESCRIPTION
**Pull request by OmgImAlexis**
A license field should probably be added to the `package.json`.
Since there are no tests I can't be sure if any of the dependencies break anything.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,16 +9,16 @@
...
-    "sqlite3": "2.1.7",
-    "step": "0.0.5",
-    "rimraf": "2.1.1",
-    "wordpress": "0.1.3",
-    "grunt": "0.3.17",
-    "grunt-wordpress": "1.2.1",
-    "grunt-check-modules": "0.1.0",
-    "grunt-jquery-content": "0.13.0",
-    "simple-log": "1.1.0"
+    "grunt": "^0.4.5",
```
Comment (mzgol) [inline package.json]: We prefer to pin dependencies so please remove the dashes (`^`).

File: README.md [hunk 2]
```diff
@@ -73,12 +73,12 @@ that content, run `grunt update`.
...
 1. `node scripts/update-server.js --console` will start the update server and
-log its output to the terminal window. This will *not* update wordpress, but
+log its output to the terminal window. This will *not* update Wordpress, but
 will let you see the result of adding a plugin locally.
 
 2. `node scripts/wordpress-update.js --console` will process the changes in
-sqlite into entries in wordpress. Note, if you're re-adding plugins that have
-already been added, you will need to remove those entries from wordpress.
+sqlite into entries in Wordpress. Note, if you're re-adding plugins that have
+already been added, you will need to remove those entries from Wordpress.
```
Comment (mzgol) [inline README.md]: It's WordPress, not Wordpress so both the old version and the new one is incorrect. Could you fix that?

**Comment by mzgol** [comment]
> A license field should probably be added to the package.json.

Agreed, it should be MIT (I think).

@scottgonzalez do we still maintain this repo or is it meant to be mostly read only?

**Comment by scottgonzalez** [comment]
This site has been dead for over a year.

**Pull request closed without merging**